frontend/src/components/templates/Login/NextSteps.js

```diff
@@ -27,25 +32,43 @@ const NextSteps = ({ targetApi, isService = false }) => {
           <div>
             <AuthIcon />
           </div>
-          <div>S’authentifier</div>
+          <div>S’identifier avec MonComptePro</div>
         </div>
         <div>
           <div>
             <DemandeIcon />
           </div>
-          <div>Remplir sa demande d’habilitation</div>
+          <div>
+            {APISDGFIP.find((targetApiID) => targetApiID === targetApi)
```

Comment:
From [developper mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) it suppose to work but  the first condition (APISDGFIP) is not working, it is displaying the default value. 

function example() {
  return condition1 ? value1
        : condition2 ? value2
        : condition3 ? value3
        : value4;
}